.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Feature request
+about: " Suggest a new feature to enhance Cantera"
```

Comment:
Leading space here, and does it need the quotes?

Reply:
Fixed the leading space. I'm leaving the quotes because GitHub put them there when I used their tool to create these issue templates.